module-2/backend/week_two.md

```diff
@@ -13,8 +14,14 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+  We can call on Team any ActiveRecord method even tough they are not describe.
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+  
+  Team.find(4).name
+  
+  Team.find(4).owner_id
+  Owner.find(ownder_id)
```

Comment:
Doesn't look like we've defined `owner_id` as a local variable anywhere in this code. You'd need to save that to a local to be able to use it here in this last line. For example:

```ruby
Team.find(4).name
owner_4_id = Team.find(4).owner_id
Owner.find(owner_4_id)
```